README.md

```
@@ -1,3 +1,5 @@
+[![MCP Toplist](https://mcptoplist.com/badge/io.github.ENTERPILOT%2Fgomodel.svg)](https://mcptoplist.com/server/io.github.ENTERPILOT%2Fgomodel)
```

markdownlint-cli2 (0.23.0)

[warning] First line in a file should be a top-level heading

(MD041, first-line-heading, first-line-h1)